plugins/spu2-x/src/RegLog.cpp

```diff
@@ -178,6 +178,7 @@ void SPU2writeLog(const char *action, u32 rmem, u16 value)
                 break;
             case REG_S_VMIXL:
                 RegLog(1, "VMIXL0", rmem, core, value);
+                break;
```

Comment:
Is this correct ? Maybe the two cases need to share 1 break.

Reply:
This is actually code for logging what the registers are doing, and there are other cases just like that that do have a break in the same switch, so I'm pretty sure it was accidentally left out.

Reply:
Cool, just making sure.